app/client.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import Navbar from './user/components/Navbar';
-import PostView from './user/post/PostView';
+import { BrowserRouter } from 'react-router-dom';
+import App from './auth/components/App';
 
 require('./globals.scss');
 
 ReactDOM.render(
-  <div className="root">
-    <Navbar />
-    <PostView />
-  </div>,
-  document.getElementById('app'),
+  (
+    <BrowserRouter>
```

Comment:
BrowserRouter does not work with Redux. Use  lower lever Router with history instead.

Reply:
The official [documentation](http://redux.js.org/docs/advanced/UsageWithReactRouter.html) of Redux for integrating with React Router has used BrowserRouter.

Reply:
I had referred to [this](https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/Router.md)
I was receiving some error with BrowserRouter. I will give it a try again, Approving the request